File: 9.java
class Solution {
    public boolean isPalindrome(int x) {
        String temp=""+x;
        for(int i=0;i<temp.length();i++){
            if(temp.charAt(i)==temp.charAt(temp.length()-i-1)){
               continue; 
            }else{
                return false;
            }
        }
        return true;
    }
}

File: 9.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return int(-self.isPalindrome(str(-x)))==x
        return int(str(x)[::-1])==x
        
